test/support/character_front_end.rb

```diff
@@ -163,7 +163,7 @@ def create_n_objects(count, factory_name)
 
     def create_per_page_plus_n_objects(n, factory_name)
       y                        =  current_window.size[1]
-      @items_per_page          = ((y+50)/60)*2
+      @items_per_page          = ((y)/60)*2
```

Comment:
Surrounding space missing for operator `*`.<br>Surrounding space missing for operator `/`.